internal/test/mock/fixtures.go

```diff
@@ -0,0 +1,45 @@
+package mock
+
+import (
+	"github.com/algorandfoundation/hack-tui/api"
+)
+
+var VoteKey = []byte("TESTKEY")
+var SelectionKey = []byte("TESTKEY")
+var StateProofKey = []byte("TESTKEY")
```

Comment:
should we use different values here? otherwise an `assert expectedVoteKey == actualVoteKey` would work for any fixture key

Reply:
They are there to test the pointers and act as a base, we also clone the fixtures in a few tests since we don't have table tests. Maybe something for the daemon?